app.js

```diff
@@ -6,12 +6,14 @@ const morgan = require("morgan");
 
 const appDataSource = require("./api/models/dataSource");
 const route = require("./api/routes");
+const { errorHandler } = require("./api/utils/error");
 
 const app = express();
 app.use(express.json());
 app.use(cors());
 app.use(morgan("tiny"));
 app.use(route);
+app.use(errorHandler);
```

Comment:
```suggestion
app.use(route)

app.all('*', (req, res, next) => {
    const err = new Error(`Can't find ${req.originalUrl} on this server!`);
    err.statusCode = 404;
    next(err);
  });

app.use(errorHandler)
```
routes에 작성되지 않은, 요청에 대해 찾을 수 없는 엔드포인트에 대하여 404 status code를 뱉을 수 있는 코드도 추가되면 좋을 것 같습니다.